buildSrc/src/main/resources/templates/returnTypeInterface.mustache

```diff
@@ -0,0 +1 @@
+{{#vendorExtensions.x-java-response-stream-type}}java.util.stream.Stream<{{vendorExtensions.x-java-response-stream-type}}>{{/vendorExtensions.x-java-response-stream-type}}{{^vendorExtensions.x-java-response-stream-type}}{{{returnType}}}{{/vendorExtensions.x-java-response-stream-type}}
```

Comment:
We can do something like:

```{{container_type}}{{^container_type}}java.util.List{{/container_type}}```

to have a default value of List so the standard behavior is retained, while still allowing us to provide our own containers for arrays.

Reply:
@Ceiu I reworked a bit, now `x-java-response-container-type` specifies the container type (leave off to get `List`.